client/src/index.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <title>My App</title>
+</head>
+
+<body>
+<div id="app" />
+
+<hr />
+<div id="brief-app" />
```

Comment:
In common all react application rendering in one tag. in your case it is 'app', and unnecessary tags are not needed here(hr, div id="brief-app")